sdk/communication/azure-communication-callingserver/pom.xml

```diff
@@ -179,7 +179,7 @@
 
   <profiles>
     <profile>
-      <id>is.jdk.11</id>
+      <id>java9plus</id>
       <activation>
         <jdk>[9,)</jdk>
       </activation>
```

Comment:
It seems some of the old -add-opens still stay in the same place, like below?

Reply:
I have another PR which is updating the argline configurations. I'll merge this first then clean up everything in the other PR.

https://github.com/Azure/azure-sdk-for-java/pull/24222